plugins/license-gather-plugin/src/main/kotlin/com/github/vlsi/gradle/license/GatherLicenseTask.kt

```diff
@@ -484,7 +485,7 @@ open class GatherLicenseTask @Inject constructor(
                 )
                 continue
             }
-            if (!file.endsWith(".jar")) {
+            if (file.extension != "jar") {
```

Comment:
This is a nice catch. It might be worth filing an IDEA ticket to https://youtrack.jetbrains.com

Reply:
Created https://youtrack.jetbrains.com/issue/KTIJ-20652